ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PurchaseButton.kt

```diff
@@ -113,6 +123,19 @@ private fun PurchaseButton(
     }
 }
 
+@ReadOnlyComposable
```

Comment:
I _think_ this is correct? 🤔

Reply:
Yeah, since this composable doesn't modify the composable tree, I think it can be a readonly composable 👍